templates/netlify/remix.init/index.js

```diff
@@ -33,12 +34,32 @@ async function copyEdgeTemplateFiles(files, rootDirectory) {
   }
 }
 
+async function updatePackageJsonForEdge(filepath) {
+  let { dependencies, ...restOfPackageJson } = JSON.parse(
+    await fs.readFile(filepath, "utf-8")
+  );
+
+  delete dependencies["@remix-run/netlify"];
+  dependencies["@remix-run/netlify-edge"] = "*";
+
+  let newPackageJson = {
+    ...restOfPackageJson,
+    dependencies,
+  };
+
+  await fs.writeFile(
+    filepath,
+    JSON.stringify(sortPackageJson(newPackageJson), null, 2) + "\n"
+  );
```

Comment:
Let's use `@npmcli/package-json` to update `package.json` instead of doing this all manually.

Reply:
Will do. I went this route based on what you linked to the other day, but if there's something less manual in place already, I'm game for that.

Reply:
I just realized what I did lol. I had implemented what was removed. 🙃  All fixed!